xclim/core/indicator.py

```diff
@@ -325,33 +329,35 @@ def update_attrs(self, ba, das):
           Input arrays.
         ba: bound argument object
           Keyword arguments of the `compute` call.
+        attrs : Mapping[str, str]
+          The attributes to format and update.
+        identifier : str
```

Comment:
I think it's confusing to use `identifier` to describe something that is not exactly the class identifier.

Reply:
Right. Could be "sub_id", or "child_id" ?